Project/dataplot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
Unused imports = CustomJS & output_notebook(in commented out block), should we remove?  Ok if no, just curious on reasoning